SRWebClient/SRWebClient.swift

```diff
@@ -108,11 +108,11 @@ public class SRWebClient : NSObject
     func build(dataDict:RequestData?) -> String? {
         var dataList: [String] = [String]()
         if(dataDict != nil) {
-            for (key, value : AnyObject) in dataDict! {
+            for (key, value) in dataDict! {
                 dataList.append("\(key)=\(value)")
             }
         }
-        return join("&", dataList).stringByAddingPercentEscapesUsingEncoding(NSUTF8StringEncoding)
+        return "&".join(dataList).stringByAddingPercentEscapesUsingEncoding(NSUTF8StringEncoding)
```

Comment:
Join wasn't working for me (Xcode 7.0.1) , "Join unavailable, use joinWithSeparator()"
```swift
(dataList).joinWithSeparator("&").stringByAddingPercentEscapesUsingEncoding(NSUTF8StringEncoding)
```
worked,  also stringByAddingPercentEscapesUsingEncoding got depreciated switched to:
```swift
(dataList).joinWithSeparator("&").stringByAddingPercentEncodingWithAllowedCharacters(NSCharacterSet.URLQueryAllowedCharacterSet())
```